### ISSUE TITLE
Cannot use template as-is for big-iq due to static values in yaml file

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-azure-arm-templates-v2#5]
## Describe the bug
 
The template example for big-iq autoscale deployed as-is (no git clone or modification of URL paths) will result in a failed big-iq licensing. The example for big-iq json template points to the https://raw.githubusercontent.com/F5Networks/f5-azure-arm-templates-v2/master/examples/autoscale/bigip-configurations/runtime-init-conf-bigiq.yaml which has this snippet for BIG-IQ parameters.

```
        myLicense:
          class: License
          licenseType: licensePool
          bigIqHost: '192.168.1.4'
          bigIqUsername: azureuser
          bigIqPassword: "{{{BIGIQ_PASSWORD}}}"
          licensePool: clpv2
          skuKeyword1: F5-BIG-MSP-BT-1G
          tenant: myTenant
          unitOfMeasure: hourly
          reachable: false
          hypervisor: azure
          overwrite: false
```

Only one value above is parameterized. Yet, the json template parameters require the following:
bigIqLicensePoolName, bigIqTenant, bigIqUtilityKey, and so on.

 
## Expected behavior
 
Runtime-init example provided yaml file should take in the BIG_IQ values I supply.
 
## Current behavior
 
The snippet of code block for license in the yaml file has static values for BIG-IQ stuff like tenant name and pool instead of parameters.
 
## Possible solution
 
Update the supplied example yaml file for big-iq to use parameterized items instead of static values.

old snippet from license block
```
        myLicense:
          class: License
          licenseType: licensePool
          bigIqHost: '192.168.1.4'
          bigIqUsername: azureuser
          bigIqPassword: "{{{BIGIQ_PASSWORD}}}"
          licensePool: clpv2
```

new
```
        myLicense:
          class: License
          licenseType: licensePool
          bigIqHost: "{{{BIGIQ_HOST}}}"
          bigIqUsername: "{{{BIGIQ_USER}}}"
          bigIqPassword: "{{{BIGIQ_PASSWORD}}}"
          licensePool: "{{{BIGIQ_LICENSE_POOL_NAME}}}"
and so on...
```

## Context
 
Trying to test autoscale with new v2 templates using BIG_IQ for licensing with limited knowledge and limited effort. Having an example to push as-is with limited effort is needed. Yes, we will need a BIG_IQ running. Yes, we need some other basic Azure knowledge. But...the current template provided yaml file is not an accurate example for those that need a little help.

**Comment by JeffGiroux** [comment]
In addition, the provided DO json file used by the the BIG-IQ deployment also has static values in the license block fro BIG-IQ items. These too need to be parameterized so as to take in the values provided by the user when kicking off the ARM json template.

https://github.com/F5Networks/f5-azure-arm-templates-v2/blob/master/examples/autoscale/bigip-configurations/autoscale-do-bigiq.json

**Comment by shyawnkarim** [comment]
Thanks for reporting these issues. This is now being tracked internally with ID ESECLDTPLT-2641.

**Comment by shyawnkarim** [comment]
Closing.

This bug was fixed in [Release 2.0.0.0](https://github.com/F5Networks/f5-azure-arm-templates-v2/releases/tag/v2.0.0.0).
